design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -0,0 +1,261 @@
+        items: [ID!]!
+    ): RemoveRequisitionListItemsOutput
+
+    "Update Items in requisition list"
+    updateRequisitionListItems(
+        "unique Id of requisition list"
+        id: ID!,
+        "Items to be updated from requisition list"
+        items: [UpdateRequisitionListItemsInput!]!
+    ): UpdateRequisitionListItemsOutput
+
+    "Add Requisition List Items To Cart"
+    addRequisitionListItemToCart(
+        "Cart Id to which requisition list needs to copied"
+        cart_id: String!,
+        "unique Id of requisition list"
+        requisition_list_id: ID!,
+        "selected requisition list items that are to be added"
+        item_ids: [ID!]!
+    ): AddRequisitionListItemToCartOutput
+
+    "Copy Items from Requisition List"
+    copyItemsFromRequisitionList(
+        "unique Id of source requisition list"
+        source_id: ID!,
+        "unique Id of destination requisition list"
+        destination_id: ID, # If null new requisition list will be created
+        "selected requisition list items that are to be copied from source"
+        item_ids: [ID!]!
+    ): CopyItemsFromRequisitionListOutput
+
+    "Move Items from Requisition List"
+    moveItemsFromRequisitionList(
+        "unique Id of source requisition list"
+        source_id: ID!,
+        "unique Id of destination requisition list"
+        destination_id: ID, # If null new requisition list will be created
+        "selected requisition list items that are to be moved from source"
+        item_ids: [ID!]!
+    ): MoveItemsFromRequisitionListOutput
+}
+
+type RemoveRequisitionListItemsOutput {
+    requisition_list : RequisitionList
+}
+
+type UpdateRequisitionListItemsOutput {
+    requisition_list : RequisitionList
+}
+
+type AddProductsToRequisitionListOutput {
+    requisition_list : RequisitionList
+}
+
+type CsvOutput {
+    "file name"
+    file_name: String
+    "data for csv file with headers and records"
+    csv_data: String!
+}
+
+input RequisitionListItemsInput {
+    sku: String!
+    quantity: Float
+    parent_sku: String
+    "selected option ID"
+    selected_options: [String!]
+    "entered Options ID"
+    entered_options: [EnteredOptionInput!]
+}
+
+input UpdateRequisitionListItemsInput {
+    "unique ID of Requisition List Item"
+    item_id: ID!
+    customizable_options: [CustomizableOptionInput]
+    quantity: Float
+}
+
+type AddRequisitionListItemToCartOutput {
+    cart: Cart # since requisition list is not mutated it is not part of the output
+}
+
+type CopyItemsFromRequisitionListOutput {
+    "Destination Requisition List"
+    list : RequisitionList # since source requisition list is not mutated it is not part of the output
+}
+
+type MoveItemsFromRequisitionListOutput {
+    "Source Requisition List"
+    source : RequisitionList
+    "Destination Requisition List"
+    destination : RequisitionList
+}
+
+"PageInfo provides navigation for the query response"
+type PageInfo {
```

Comment:
There's already a type for representing paging with an identical structure, see `SearchResultPageInfo`

Reply:
sure I will use that.